pkg/services/accesscontrol/models.go

```diff
@@ -310,9 +310,6 @@ const (
 	ActionLicensingUpdate      = "licensing:update"
 	ActionLicensingDelete      = "licensing:delete"
 	ActionLicensingReportsRead = "licensing.reports:read"
-
-	// Team actions
-	ActionTeamsCreate = "teams:create"
```

Comment:
As mentioned in the other comment, we could actually define all actions here instead of putting them in the api package.